code/index.html

```diff
@@ -1,30 +1,254 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8" />
+    
+  <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
+    <title>Dream cakes 🎂</title>
+    <link rel="stylesheet" href="style.css">
+  </head><body>
 
+  <header>
+    <div class="overlay">
+      <h1>Dream Cakes</h1>
+      <p class="tagline">Create your own dream cake<br></p>
+      <button class="btn">Contact us</button>
+    </div>
+   
+        
+    <div class="video1">
+      <video autoplay muted loop>
+      <source src="./videos/CakeVideo.mp4" type="video/mp4">
+      </video>
+    </div>
+
+    <div class="video2">
+      <video autoplay muted loop>
+      <source src="./videos/CakeVideo2.mp4" type="video/mp4">
+      </video>
+    </div>
+
+    <div class="video3">
+      <video autoplay muted loop>
+      <source src="./videos/CakeVideo3.mp4" type="video/mp4">
+      </video>
+    </div>
+
+  </header>
+
+    
     <!-- Signup form -->
+  <section class="content">
+    <form 
+    class="form"
+    action="http://httpbin.org/anything"
+    method="POST"
+    >
+
+    <h2>Get your dream cake</h2>
+    <p>Choose a cake style from the list to the
+       right and we will send you an offer</p>
+    
+    <div class="form-group">
+      <label for="firstName">First name*</label>
+      <input
+      id="firstName"
+      type="text"
+      placeholder="Type your first name"
+      name="firstNameValue"
+      required
+      />
+    </div>
+
+    <div class="form-group">
+      <label for="lasttName">Last name*</label>
+      <input
+      id="lasttName"
+      type="text"
+      placeholder="Type your last name"
+      name="lastNameValue"
+      required
+      />
+    </div>
+
+    <div class="form-group">
+      <label for="email">Email*</label>
+      <input
+      id="email"
+      type="email"
+      placeholder="Type your e-mail"
+      name="emailValue"
+      required
+      />
+    </div>
+
+    <div class="form-group">
+      <label for="message">What cake style do you want?</label><br>
+      <textarea
+      id="message"
+      cols="30"
+      rows="2"
+      maxlength="100"
+      name="messageValue"
+      placeholder="Max. 100 characters"
+      required
+       ></textarea>
+    </div>
+
+      <label>How many pieces of cake?</label>
+
+    <div class="form-group">
+      <input 
+      type="radio" 
+      id="8" 
```

Comment:
You did connection `input`-`form` very well, but you should avoid naming your `id`s by numbers only. You can add numbers to you `id` name, but they shouldn't start from it. So, you can name it `radio8` or `radio8element`, but not `8`, `8radio` or `8radioelement`